browser-test/src/support/admin_questions.ts

```diff
@@ -163,15 +163,26 @@ export class AdminQuestions {
     expect(tableInnerText).not.toContain(questionName)
   }
 
-  async gotoQuestionEditPage(questionName: string) {
+  private async gotoQuestionEditOrNewVersionPage(
+    questionName: string,
```

Comment:
Not blocking and a style nit: Since there are two arguments of the same type, it's pretty easy to get argument order mixed up. What about destructuring so that callers need to pass in "named arguments"?

Reply:
Done.